client/my-sites/site-settings/navigation.jsx

```diff
@@ -46,9 +46,7 @@ export default React.createClass( {
 
 	getExportPath() {
 		const { site } = this.props;
-		return site.jetpack
-			? `${ site.options.admin_url }export.php`
-			: `/settings/export/${ site.slug }`;
+		return `/settings/export/${ site.slug }`;
```

Comment:
If we're not doing anything interesting in this function, we could maybe just inline this in the render method.